aws-iot-device-sdk-java/src/test/java/com/amazonaws/services/iot/client/system_props_websocket.properties

```diff
@@ -0,0 +1,7 @@
+clientEndpoint=ajje7lpljulm4-ats.iot.us-east-1.amazonaws.com
+clientId=DefaultClientId
+thingName=DefaultThingName
+privateMaterial=arn:aws:secretsmanager:us-east-1:123124136734:secret:V1IotSdkIntegrationTestWebsocketSecretAccessKey-MKTSaV
```

Comment:
Question: Does the `ClientEndpoint` property in this file need to be in a secret?

Reply:
It should be fine, we could not connect to endpoint without the correct authorization. Of cause we can always do it in a safer way and save it on the aws secret manager.

Reply:
Ah okay, then I'd say it's fine then. I just was not sure and wanted to ask.